src/verifyEmail.js

```diff
@@ -0,0 +1,27 @@
+function verifyEmail(appId, serverURL) {
```

Comment:
If you look at it right now - it could a simply function, sure, but as we are trying to decouple and move away from using globals - the `DatabaseAdapter` piece as a first line in the function is actually a global, and should be injected at construction time of a controller - since it is required for it to function.
The idea is to have a single DatabaseAdapter per server, which is what we allow right now, and stop doing things like `getDatabaseConnection` from a global.

Which at this point will make sense to move this entire piece into `UsersController`, which will make it much simpler to extend.
Also, we might as well split some logic that deals with request/response here into UsersPublicRouter (because you don't need app specific security and friends).